src/__test__/node-test.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
```

Comment:
This file should be named node.test.js - Jest only looks for files with the .test.js extension after it. That's why your tests aren't running.